### DISCUSSION TITLE
How can compatable work pnpm with asdf?

### DISCUSSION THREAD

**Discussion by offwork** [pnpm/pnpm#7843] [Q&A]
How can compatible work pnpm with asdf?

I work with different node package manager like yarn or npm at many projects so I use asdf-vm for node version. In short, I don't need pnpm's nodejs, Pnpm even causes confusion in many cases. _I have to admit, maybe I could do this with pnpm, but I don't want to opt for that._

So, how can I work with asdf nodejs with without nodejs from pnpm?

**Comment by weyert** [comment]
`asdf-pnpm` works pretty well for me? I am using [1] together with the `nodejs` asdf plugin

[1] https://github.com/jonathanmorley/asdf-pnpm

**Reply by offwork** [reply]
Yes, I saw this plugin but I wasn't sure to use it. thank you very much for your advice.
